src/Confluent.Kafka/AdminClient.cs

```diff
@@ -501,6 +501,17 @@ public Metadata GetMetadata(string topic, TimeSpan timeout)
         public int AddBrokers(string brokers)
             => kafkaHandle.AddBrokers(brokers);
 
+        /// <summary>
+        ///     Refer to <see cref="Confluent.Kafka.IClient.OauthBearerSetToken(string, long, string, IDictionary{string, string})" />
+        /// </summary>
+        public void OauthBearerSetToken(string tokenValue, long lifetimeMs, string principalName, IDictionary<string, string> extensions)
+            => kafkaHandle.OauthBearerSetToken(tokenValue, lifetimeMs, principalName, extensions);
+        
+        /// <summary>
+        ///     Refer to <see cref="Confluent.Kafka.IClient.OauthBearerSetTokenFailure(string)" />
+        /// </summary>
+        public void OauthBearerSetTokenFailure(string errstr)
```

Comment:
i think the param name should probably be just `error`, `errstr` isn't very idiomatic c#